challenger/src/duplex_challenger.rs

```diff
@@ -106,8 +106,8 @@ where
     F: Copy,
     P: CryptographicPermutation<[F; WIDTH]>,
 {
-    fn observe(&mut self, valuess: Vec<Vec<F>>) {
-        for values in valuess {
+    fn observe(&mut self, values: Vec<Vec<F>>) {
+        for values in values {
```

Comment:
Please do not name the item variable the same as the container variable name

Reply:
thks to point out, fixed

Reply:
@jorgeantonio21 Done